lib/MusicBrainz/Server/Entity/URL/JunoDownload.pm

```diff
@@ -0,0 +1,26 @@
+package MusicBrainz::Server::Entity::URL::JunoDownload;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name {
+    my $self = shift;
+
+    return "Juno Download";
+}
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright (C) 2018 MetaBrainz Foundation
```

Comment:
_You should replace that with your own name (unless you intended to assign copyright to MeB)._

Reply:
Good catch, updated.